src/middlewared/middlewared/plugins/virt/stats.py

```diff
@@ -33,18 +30,22 @@ def run_sync(self):
                 else:
                     break
 
-            data = {}
-            if not bool(netdata_metrics):
-                data['error'] = True
-                data['errname'] = 'FAILED_TO_CONNECT'
-            else:
-                data.update(get_cgroup_stats(netdata_metrics, [instance_id])[instance_id])
-                data['error'] = False
-                data['errname'] = None
+            data = {
+                'error': True,
+                'errname': 'FAILED_TO_CONNECT',
```

Comment:
I just removed this paradigm of showing "FAILED_TO_CONNECT" in https://github.com/truenas/middleware/pull/16145 because the UI has never used it. They actually had issues with this type of response because they don't guard against it. So we need to remove this here as well.